src/org/usfirst/frc4904/robot/commands/IntakeRollers.java

```diff
@@ -0,0 +1,24 @@
+package org.usfirst.frc4904.robot.commands;
+
+import org.usfirst.frc4904.robot.subsystems.CubeIO;
+import edu.wpi.first.wpilibj.command.Command;
+
+
+public class IntakeRollers extends Command {
```

Comment:
I think @HowardStark meant `RollersIntake` but I agree with the reasoning.

Reply:
@Krakavius I'd actually disagree with @HowardStark (though he can correct me) and say that maybe this could be more specific. Because there are multiple intakes, maybe write "RollersCubeIO"? IDK but that's my preference.

Reply:
No, this is fine as there will likely only be one intake at the end, and that specificity should come at the branch-name level.

Reply:
@ajnadel that was what i meant.